### PR TITLE
GHC-77: Update labs

### PR DESCRIPTION
**Pull request by nuhuhmutebi**

### CODE REVIEW

File: configuration/concepts/lab_tests.csv [hunk 1]
```diff
@@ -1,24 +1,110 @@
...
+09db95e4-185c-4a22-9464-10c77dff27be,,,Urine,Orina,Orina,Urine,,Sample,N/A,,,Urinalysis; Urine pregnancy test; Urine culture; PH; Specific gravity; Glucose; Ketones; Nitrites; Leukocyte esterase; Bilirubin; Blood in urine; Protein,,,,,,,,
+faabf960-2025-42e1-8eed-f7825cfa6e2d,,,Swab,Torunda,Torunda,Swab,,Sample,N/A,,,Wet Mount; Wound culture; Chlamydia; ab6588f6-87ee-45ca-a6c5-dc1337f83469; 74ab27c4-c745-45ff-b7e6-e8dbaf933458; b2a75309-4eb5-414d-9f29-8ecb4ac7dd32; Yeast; Bacteria; Clue cells; Yeast; RBCs; Trichomonas; Ferning; Whiff Test,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: @nuhuhmutebi Do we need to have two Yeast tests?

File: configuration/concepts/lab_tests.csv [hunk 2]
```diff
@@ -1,24 +1,110 @@
...
+8696ef92-4b42-45a1-b4a2-5e860b6bc311,,,AST,AST,AST,AST,,LabTest,Numeric,,,,,,,,,,,
+5ade6005-066c-434b-b4b6-639ebfe45f50,,,ALP,ALP,ALP,ALP,,LabTest,Numeric,,,,,,,,,,,
+d225f0c6-5215-476c-974b-c8fc58531c1d,,,Total Cholesterol,Colesterol total,Colesterol total,Total Cholesterol,,LabTest,Numeric,,,,,,,,,,,
+84a5df72-4916-4f92-9de7-836794950828,,,HDL,HDL,HDL,HDL,,LabTest,Numeric,,,,,,,,,,,
+14896245-7961-4f74-8119-28156b028a4e,,,LDL,LDL,LDL,LDL,,LabTest,Numeric,,,,,,,,,,,
+93f3addd-edcc-4b44-ba2e-c1419a721940,,,Triglycerides,Triglicéridos,Triglicéridos,Triglycerides,,LabTest,Numeric,,,,,,,,,,,
+fe1619a0-6171-41ce-b8ba-b0c4c47d880b,,,PT,PT,PT,PT,,LabTest,Numeric,,,,,,,,,,,
+1fde01d8-7c48-407d-80ff-6413a6211b95,,,aPTT,aPTT,aPTT,aPTT,,LabTest,Numeric,,,,,,,,,,,
+ce601386-f54c-4ccc-bb39-ce1f07a8abe9,,,INR,INR,INR,INR,,LabTest,Numeric,,,,,,,,,,,
+3ea1a370-b718-4122-942f-a92499e2fad8,,,C Reactive Protein,Proteína C-reactiva,Proteína C-reactiva,C Reactive Protein,,LabTest,Numeric,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: Lets have this as `C-Reactive Protein`

**Review by reagan-meant — CHANGES_REQUESTED** [review]

**Comment by nuhuhmutebi** [comment]
All Comments Addressed check commit above.
Thanks @reagan-meant

File: configuration/concepts/lab_tests.csv (hunk 2 above)
Comment (nuhuhmutebi) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 1 above)
Comment (nuhuhmutebi) [reply configuration/concepts/lab_tests.csv]: Done

**Comment by reagan-meant** [comment]
Looks good..Thanks @nuhuhmutebi

File: configuration/concepts/lab_tests.csv [hunk 3]
```diff
@@ -1,24 +1,110 @@
...
+b7556cb3-9e65-4efa-956a-10cf9094eaf5,,,Bilirubin,Bilirrubina,Bilirrubina,Bilirubin,,LabTest,Coded,,Negative; Small; Moderate; Large,,,,,,,,,
+9097c71d-d20e-4d12-980f-008b6c1ab9c9,,,Blood in urine,Sangre en la orina,Sangre en la orina,Blood in urine,,LabTest,Coded,,Negative; Trace; Small; Moderate; Large,,,,,,,,,
+fa14fdf3-d0df-4298-8677-f84e87467858,,,Protein,Proteína,Proteína,Protein,,LabTest,Coded,,Normal; Trace; One plus; Two plus; Three plus; Four plus,,,,,,,,,
+a24afd60-b6b6-4b7e-82b4-dc599ce62ead,,,Rapid HIV,VIH rápido,VIH rápido,Rapid HIV,,LabTest,Coded,,Positive; Negative;,,,,,,,,,
+dbff115d-32df-4bd5-8265-832e1f3cbe5e,,,Fasting blood glucose,Glucosa en sangre en ayunas,Glucosa en sangre en ayunas,Fasting blood glucose,,LabTest,Numeric,,,,Yes,mg/dL,70,126,,,,
+3a7aefbf-3334-4add-88a0-42ee1c6fd8dd,,,Random blood glucose,Glucosa en sangre aleatoria,Glucosa en sangre aleatoria,Random blood glucose,,LabTest,Numeric,,,,Yes,mg/dL,70,180,,,,
+df5686c1-ad0c-4deb-9919-f77446fd5bb9,,,HbA1C,HbA1C,HbA1C,HbA1C,,LabTest,Numeric,,,,Yes,%,4,6.5,,,,
+1e3e425c-eee2-4d61-8652-dd20239ec159,,,Hemoglobin,Hemoglobina,Hemoglobina,Hemoglobin,,LabTest,Text,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: Let us make this Numeric

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/lab_tests.csv (hunk 3 above)
Comment (nuhuhmutebi) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv [hunk 4]
```diff
@@ -1,24 +1,110 @@
...
+2b2f6511-d428-4c25-86af-f015b495d613,,,Bacteria,Bacterias,Bacterias,Bacteria,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+0a4eafae-296c-4ba6-aef4-a843337aa29b,,,Clue cells,Células de pista,Células de pista,Clue cells,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+d3966267-5882-437b-ae14-18dee9657d14,,,RBCs,Hematíes,Hematíes,RBCs,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+1b985633-13de-4afa-828e-ef38721c660e,,,Ferning,Helecho,Helecho,Ferning,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+dbb2a948-ab68-4474-a24b-5f0ecf697f88,,,Whiff Test,Prueba de olor,Prueba de olor,Whiff Test,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+acdc2f09-2844-43e0-9067-8e71da8319ae,,,Urinalysis,Análisis de orina,Análisis de orina,Urinalysis,,LabSet,N/A,,,PH; Specific gravity; Glucose; Ketones; Nitrites; Leukocyte esterase; Bilirubin; Blood in urine; Protein,,,,,,,,
+4bb7df94-7b92-4e5f-8221-064a76d65e17,,,Yeast,Hongos,Hongos,Yeast,,LabTest,Text,,,,,,,,,,,
+b2a75309-4eb5-414d-9f29-8ecb4ac7dd32,,,Bacterial vaginosis,Vaginosis bacteriana,Vaginosis bacteriana,Bacterial vaginosis,,LabTest,Text,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: We make this coded with the answers as `Positive;Negative`

File: configuration/concepts/lab_tests.csv [hunk 5]
```diff
@@ -1,24 +1,110 @@
...
+334cf7c5-033c-4c0c-b605-7f96c983b51d,,,Fecal occult,Oculto fecal,Oculto fecal,Fecal occult,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+2b2f6511-d428-4c25-86af-f015b495d613,,,Bacteria,Bacterias,Bacterias,Bacteria,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+0a4eafae-296c-4ba6-aef4-a843337aa29b,,,Clue cells,Células de pista,Células de pista,Clue cells,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+d3966267-5882-437b-ae14-18dee9657d14,,,RBCs,Hematíes,Hematíes,RBCs,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+1b985633-13de-4afa-828e-ef38721c660e,,,Ferning,Helecho,Helecho,Ferning,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+dbb2a948-ab68-4474-a24b-5f0ecf697f88,,,Whiff Test,Prueba de olor,Prueba de olor,Whiff Test,,LabTest,Coded,,Positive; Negative,,,,,,,,,
+acdc2f09-2844-43e0-9067-8e71da8319ae,,,Urinalysis,Análisis de orina,Análisis de orina,Urinalysis,,LabSet,N/A,,,PH; Specific gravity; Glucose; Ketones; Nitrites; Leukocyte esterase; Bilirubin; Blood in urine; Protein,,,,,,,,
+4bb7df94-7b92-4e5f-8221-064a76d65e17,,,Yeast,Hongos,Hongos,Yeast,,LabTest,Text,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: We make this coded with the answers as `Positive;Negative`

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/lab_tests.csv (hunk 5 above)
Comment (nuhuhmutebi) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 4 above)
Comment (nuhuhmutebi) [reply configuration/concepts/lab_tests.csv]: Done

**Pull request merged**